frequency-plans.yml

```diff
@@ -2,104 +2,121 @@
   name: Australia 915-928 MHz, FSB 1
   description: Default frequency plan for Australia, using sub-band 1
   base-frequency: 915
+  country-codes: [br, cl, nz, ar, au]
   file: AU_915_928_FSB_1.yml
 
 - id: AU_915_928_FSB_2
   name: Australia 915-928 MHz, FSB 2 (TTN)
   description: TTN Community Network frequency plan for Australia, using sub-band 2
   base-frequency: 915
+  country-codes: [br, cl, nz, ar, au]
   file: AU_915_928_FSB_2.yml
 
 - id: AU_915_928_FSB_6
   name: Australia 915-928 MHz, FSB 6
   description: Frequency plan for Australia, using sub-band 6, which overlaps with Asia 923-925 MHz
   base-frequency: 915
+  country-codes: [br, cl, nz, ar, au]
   file: AU_915_928_FSB_6.yml
 
 - id: CN_470_510_FSB_11
   name: China 470-510 MHz, FSB 11
   description: TTN Community Network frequency plan for China, using sub-band 11
   base-frequency: 470
+  country-codes: [cn]
   file: CN_470_510_FSB_11.yml
 
 - id: AS_920_923
   name: Asia 920-923 MHz
   description: TTN Community Network frequency plan for Asian countries, using frequencies ≤ 923 MHz
   base-frequency: 915
+  country-codes: [my, sg]
   file: AS_920_923.yml
 
 - id: AS_920_923_LBT
   name: Asia 920-923 MHz with LBT
   description: TTN Community Network frequency plan for Asian countries, using frequencies ≤ 923 MHz with listen-before-talk
   base-frequency: 915
   base-id: AS_920_923
+  country-codes: [jp, my, sg]
   file: lbt_80_over_128.yml
 
 - id: AS_923_925
   name: Asia 923-925 MHz
   description: TTN Community Network frequency plan for Asian countries, using frequencies ≥ 923 MHz
   base-frequency: 915
+  country-codes: [bn, kh, hk, id, la, tw, th, vn]
   file: AS_923_925.yml
 
 - id: AS_923_925_LBT
   name: Asia 923-925 MHz with LBT
   description: TTN Community Network frequency plan for Asian countries, using frequencies ≥ 923 MHz with listen-before-talk
   base-frequency: 915
   base-id: AS_923_925
+  country-codes: [bn, kh, hk, id, la, tw, th, vn]
   file: lbt_80_over_128.yml
 
 - id: AS_923_925_TTN_AU
   name: Asia 923-925 MHz (TTN Australia)
   description: TTN Community Network frequency plan for Asia 923-925 MHz in Australia
   base-frequency: 915
   base-id: AS_923_925
+  country-codes: [au]
   file: AS_923_925_TTN_AU.yml
 
 - id: KR_920_923_TTN
   name: South Korea 920-923 MHz
   description: TTN Community Network frequency plan for South Korea
   base-frequency: 915
+  country-codes: [kr]
   file: KR_920_923_TTN.yml
 
 - id: IN_865_867
   name: India 865-867 MHz
   description: Default frequency plan for India
   base-frequency: 868
+  country-codes: [in]
   file: IN_865_867.yml
 
 - id: RU_864_870_TTN
   name: Russia 864-870 MHz
   description: TTN Community Network frequency plan for Russia
   base-frequency: 868
+  country-codes: [ru]
   file: RU_864_870_TTN.yml
 
 - id: ISM_2400_3CH_DRAFT2
   name: LoRa 2.4 GHz with 3 channels draft 2
   description: LoRa 2.4 GHz plan with 3 channels for global use draft 2
   base-frequency: 2450
+  country-codes: []
```

Comment:
Can we do a wildcard here or something?

Reply:
When I first started thinking about the issue, I had the idea of supporting aliases (`_eu_` and `_all_`), but in the end I think it's better to be explicit.

So either we list all country codes here, or we decide that `country-codes` is an optional field that restricts the list of applicable countries if set.

Reply:
> So either we list all country codes here, or we decide that `country-codes` is an optional field that restricts the list of applicable countries if set.

I'm good with either

Reply:
every country allows 2.4ghz? do we know that because wifi works everywhere, or is it documented officially somewhere

Reply:
It is intended to be globally allowed, indeed. Semtech looked at all regulating bodies (FCC, ETSI, China, etc), and ISM2400 is designed to be compatible by considering the lowest common denominator.

Reply:
updated with all countries from https://en.wikipedia.org/wiki/List_of_ISO_3166_country_codes